### ISSUE TITLE
Group ID still prompts for enrollment

### ISSUE THREAD

**Issue by Kos33** [helmlingp/apps_WS1UEMWin10Migration#3]
HEy Phil,
I wanted to let you know that upon using the intunetows1win10migration.ps1 I get a successful deployment, the intelligent hub installs and enrolls the device but requires the user to put in the Group ID inside the GUI even though I am using the parameters

I noticed the flag was -OGName and the command line flags for the msi argument actually calls for the -LGName in the command line enrollment for windows 10

Upon updating the deployfiles.ps1 and intunetows1win10migration.ps1 from OGname to LGname in all places, it does not work as expected with an error in the scheduled task 0x1

Wondering if this is expected or if I am missing something.

**Comment by helmlingp** [comment]
Hi Kos33,
the prompt is due to the way the enrolment flow works with Intelligent Hub. The enrolment with staging credentials uses the OGName to stage the enrolment, but then needs a user to "switch" to. Depending on the WS1 Access authentication policy, this prompt can be hidden. We effectively would enable single sign on to WS1. The best way to do this is to deploy a user certificate with the user's username or UPN in the certificate which needs to match a user in the Active Directory configured in UEM.
The LGName and OGName are the same thing, LGName is the old name for OGName used with the command line enrolment process. Do not change this. This is why the process fails.

Hope that makes sense

**Comment by Kos33** [comment]
Thanks Phil,
This makes sense. We were attempting to reduce the user interaction. The group ID prompt is kind of a bummer.  The parameters ask for the staging account server & Group ID but we are still prompted for the group ID... This is before the user can authenticate (switch over) to the access policies. 

Thanks!

**Comment by Kos33** [comment]
Hey Phil,
I understand the group ID prompt will happen when switching. I did test with a directory account and that did not work. .

We also tried Group ID mapping based on the user group thinking that would bypass the group ID prompt. Our testing did not work and it is still prompting for the group ID... would it be possible to work through this in any way? 

<img width="954" alt="Screen Shot 2022-10-14 at 2 58 30 PM" src="https://user-images.githubusercontent.com/43039757/195931613-4723b7fe-efb5-4959-a8a7-3e3b8f29155e.png">
